test/In.ProjectEKA.HipServiceTest/DataFlow/Builder/TestBuilder.cs

```diff
@@ -18,5 +19,30 @@ internal static HealthInformationRequest HealthInformationRequest(string transac
                 new HiDataRange(faker.Random.Hash(), faker.Random.Hash()),
                 faker.Random.Hash());
         }
+
+        internal static Faker<ConsentArtefact> ConsentArtefact()
+        {
+            return new Faker<ConsentArtefact>();
+        //     var consentPurpose = new ConsentPurpose("Encounter", "ENCOUNTER","Some");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4963271)